api/index.js

```diff
@@ -11,15 +11,36 @@ var app = express();
 
 const development = process.env.NODE_ENV !== 'production';
 const iberaServicesEndpoint = config.IBERA_SERVICES_ENDPOINT;
+const documentServicesEndpoint = config.DOCUMENT_SERVICES_ENDPOINT;
 
 process.env.NODE_TLS_REJECT_UNAUTHORIZED = "0";
 
+async function getUserId(userToken){
+  try {
+    var uri = documentServicesEndpoint + `/api/user`;
+    var result = await request({
+      method: 'POST',
+      uri,
+      body: { 
+        token : userToken
+      },
+      json: true
+    });
+    console.log(`got response: ${util.inspect(result)}`);
+    return result;
+  }
+  catch (err) {
+    return res.status(HttpStatus.INTERNAL_SERVER_ERROR).json({ error: `could not retrieve userId from user token` });
```

Comment:
I would add the userToken to the error, or console.error it before sending the error back to the caller